src/seedu/addressbook/AddressBook.java

```diff
@@ -44,7 +45,7 @@
     /**
      * A decorative prefix added to the beginning of lines printed by AddressBook
      */
```

Comment:
Good! Variables should generally be local to a class. 
However, constants that are of used across the application (across many classes) should be consolidated in a single class as static final variables with public access.